intellij/src/saros/intellij/eventhandler/filesystem/LocalFilesystemModificationHandler.java

```diff
@@ -662,7 +686,7 @@ private void generateFileMove(
 
       editorManager.replaceAllEditorsForPath(oldFilePath, newFilePath);
 
-      annotationManager.updateAnnotationPath(oldFilePath.getFile(), newFilePath.getFile());
+      annotationManager.updateAnnotationPath(oldSarosFile, newSarosParent);
```

Comment:
Needs to be the new file (`newFilePath`, instantiated in l.672) and not the parent folder (which is currently used).

Reply:
Yes right...fixed